### PR TITLE
Transfer to dummy global namespace

### PR DESCRIPTION
**Pull request by AlexFreik**
- got rid of our parsing and transfer execution in global namespace
- refactored backend nodes, and separated `visible`, `position`, and etc. in one field `uidata`.
- right now frontend is still not updated, since i decided to merge your changes first
- add name field to the graph

### CODE REVIEW

File: backend/AtlasBackend/src/AtlasGraph/src/utils/kernels/execution/Executer.jl [hunk 1]
```diff
@@ -0,0 +1,23 @@
+module Executer
```
Comment (roman-timo) [inline backend/AtlasBackend/src/AtlasGraph/src/utils/kernels/execution/Executer.jl:1]: not the best name for this module ("палач") :)))
maybe "processor" or "doer" or "performer"... or etc.

**Review by roman-timo — CHANGES_REQUESTED** [review]
did you add new property "name" into AtlasGraph to return?

File: backend/AtlasBackend/src/AtlasGraph/src/utils/kernels/execution/Executer.jl (hunk 1 above)
Comment (AlexFreik) [reply backend/AtlasBackend/src/AtlasGraph/src/utils/kernels/execution/Executer.jl:1]: > did you add new property "name" into AtlasGraph to return?

did you read my description?

Screenshot 2022-06-11 at 16 07 54

**Comment by AlexFreik** [comment]
> did you add new property "name" into AtlasGraph to return?

did you read my description?

Screenshot 2022-06-11 at 16 08 42

File: backend/AtlasBackend/src/AtlasGraph/src/utils/kernels/execution/Executer.jl (hunk 1 above)
Comment (AlexFreik) [reply backend/AtlasBackend/src/AtlasGraph/src/utils/kernels/execution/Executer.jl:1]: I think it makes the name even better, haha. It executes nodes, so it is executer. `Execute` is a common name for this. `Jupyter`, `Python` uses this keyword for this, so we can also use it. Ofc, i can rename it into `Evaluator`, but i think `execute` is more consistent

Comment (AlexFreik) [reply backend/AtlasBackend/src/AtlasGraph/src/utils/kernels/execution/Executer.jl:1]: Also "палач" is execut**o**r, not `executer`

Comment (roman-timo) [reply backend/AtlasBackend/src/AtlasGraph/src/utils/kernels/execution/Executer.jl:1]: ok))

**Review by roman-timo — APPROVED** [review]

**Review by roman-timo — APPROVED** [review]
good we got into native Julia :)

**Pull request merged**